schutzbot/Jenkinsfile

```diff
@@ -13,31 +13,49 @@ pipeline {
                     agent { label "fedora31" }
                     environment { TEST_TYPE = "base" }
                     steps {
-                        sh "schutzbot/run_tests.sh"
+                        run_tests('fedora31-base')
                     }
                 }
                 stage('Fedora 31 image') {
                     agent { label "fedora31" }
                     environment { TEST_TYPE = "image" }
                     steps {
-                        sh "schutzbot/run_tests.sh"
+                        run_tests('fedora31-image')
                     }
                 }
                 stage('Fedora 32 base') {
                     agent { label "fedora32" }
                     environment { TEST_TYPE = "base" }
                     steps {
-                        sh "schutzbot/run_tests.sh"
+                        run_tests('fedora32-base')
                     }
                 }
                 stage('Fedora 32 image') {
                     agent { label "fedora32" }
                     environment { TEST_TYPE = "image" }
                     steps {
-                        sh "schutzbot/run_tests.sh"
+                        run_tests('fedora32-image')
                     }
                 }
             }
         }
     }
 }
+
+// Set up a function to hold the steps needed to run the tests so we don't
+// need to copy/paste the same lines over and over above.
+void run_tests(test_slug) {
+
+    // Run the tests from the repository.
+    sh "schutzbot/run_tests.sh"
```

Comment:
Does this function continue executing if this line fails?

Reply:
@larskarlitski https://www.jenkins.io/doc/book/pipeline/syntax/#post 🤦‍♂️

Good catch. I'll update the PR.

Reply:
Heh, I figured it's most important to see the logs when the test fails. Thanks for this!